python/ray/_private/services.py

```diff
@@ -290,6 +280,123 @@ def find_redis_address_or_die():
     return redis_addresses.pop()
 
 
+def find_gcs_address():
```

Comment:
Most of the logic is duplicated from `find_redis_address()`. Alternatively I can make that function accept a parameter for `--redis-address` vs `--gcs_address`.